src/Xamarin.Android.Tools.AndroidSdk/JdkInfo.cs

```diff
@@ -287,6 +287,7 @@ public static IEnumerable<JdkInfo> GetKnownSystemJdkInfos (Action<TraceLevel, st
 			return GetEnvironmentVariableJdks ("JI_JAVA_HOME", logger)
 				.Concat (GetWindowsJdks (logger))
 				.Concat (GetConfiguredJdks (logger))
+				.Concat (GetMacOSMicrosoftOpenJdks (logger))
 				.Concat (GetMacOSMicrosoftJdks (logger))
```

Comment:
We won't need this anymore either, correct? FWIK, this is for finding the Xamarin build of the JDK, which will no longer be shipped.

Reply:
While true, I'm uncertain about the "compatibility", e.g. if someone installs a "newer" Xamarin.Android .vsix atop their "older" Visual Studio install.

Though even there, the odds are *really high* that `GetWindowsJdks()` > `GetPreferredJdkPaths()` (which hits the registry on Windows) or > `GetConfiguredJdks()` (which on macOS uses `monodroid-config.xml`) will return a value before `GetMacOSMicrosoftJdks()` will be encountered…

Thus sure, we can remove this.